package.json

```diff
@@ -26,7 +26,7 @@
     "integration-node-encrypt": "npm run build; lerna run integration_node --stream --no-prefix -- -- encrypt -m $npm_package_config_encryptManifestList -k $npm_package_config_encryptKeyManifest -o $npm_package_config_decryptOracle",
     "integration-node": "run-s integration-node-*",
     "integration": "run-s integration-*",
-    "test_conditions": "./util/bootstrap_tsconfig 23"
+    "test_conditions": "./util/bootstrap_tsconfig 2"
```

Comment:
What does this option do?

Reply:
I am trying to get rid of it.  I just lets the test not fail if there are "exceptions" I want to remove it so that no one even sees it and thinks, "Hey, I can get aways with not writing a test by setting this..."  But as I get to that point, I want to have it running...